tools/docker/22.04/Dockerfile

```diff
@@ -8,8 +8,8 @@ ARG DEBIAN_FRONTEND=noninteractive
 # (see https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners).
 # This is to prevent the docker image getting build on a newer architecture and then failing when it
 # is loaded on an older one.
-ENV CFLAGS "-march=haswell"
-ENV CXXFLAGS "-march=haswell"
+ENV CFLAGS="-march=haswell"
+ENV CXXFLAGS="-march=haswell"
```

Comment:
For the 22.04 image I'd be tempted to bump this up to a newer system.  But we'd have to lookup what the oldest system github CI is running these days.  Not to solve in this PR, just a note.